deptofdefense/move.mil
# Service-specific information pages redesign

## Checklist

I have…


## Summary of Changes

This pull request implements the service-specific information redesign (issue #141)
- Separate pages for each branch, e.g. /service-specific-information/army
- Reads customer service info from the DB instead of putting it in code
- I didn't bother adding JS to avoid duplication of the sidebar <--> accordion component. Shared css with the one on the home page though.
- Still waiting on some updated content for navy

## Testing

To verify the changes proposed in this pull request…
clone this repo
git checkout customer-service
set up development dependencies according to CONTRIBUTING.md,
run bin/rails server, and
load up http://localhost:3000/service-specific-information
- click through each of the branches
- click links in the homepage service emblems list
- click links in the retirees page

## Screenshots
<details><summary>Wide screen army</summary>

![screencapture-localhost-3000-service-specific-information-army-1512063685592](https://user-images.githubusercontent.com/29409348/33445816-0f6d3158-d5cc-11e7-9539-1494f1f2e6ff.png)

</details>

<details><summary>Wide screen marines</summary>

![screencapture-localhost-3000-service-specific-information-marine-corps-1512063822141](https://user-images.githubusercontent.com/29409348/33445833-20cd6954-d5cc-11e7-903c-59cba25ff3fb.png)

</details>

<details><summary>Wide screen navy</summary>

![screencapture-localhost-3000-service-specific-information-navy-1512063845078](https://user-images.githubusercontent.com/29409348/33445850-2cf5b92a-d5cc-11e7-8082-e268cda0e826.png)

</details>

<details><summary>Wide screen air force</summary>

![screencapture-localhost-3000-service-specific-information-air-force-1512063869114](https://user-images.githubusercontent.com/29409348/33445867-3df9ecdc-d5cc-11e7-8870-695eb0c35900.png)

</details>

<details><summary>Wide screen coast guard</summary>

![screencapture-localhost-3000-service-specific-information-coast-guard-1512063900898](https://user-images.githubusercontent.com/29409348/33445880-4b709046-d5cc-11e7-9a1f-f3d19a589c19.png)
</details>

<details><summary>Narrow screen with banner</summary>

![screencapture-localhost-3000-service-specific-information-army-1512063924645](https://user-images.githubusercontent.com/29409348/33446088-f20fa4b4-d5cc-11e7-8959-969cae3417cc.png)

</details>

<details><summary>Narrow screen with customer service expanded</summary>

![screencapture-localhost-3000-service-specific-information-army-1512063941176](https://user-images.githubusercontent.com/29409348/33446089-f220d888-d5cc-11e7-937e-b38430dc1430.png)

</details>

<details><summary>Narrow screen with no banner</summary>

![screencapture-localhost-3000-service-specific-information-marine-corps-1512064103399](https://user-images.githubusercontent.com/29409348/33446090-f230ea98-d5cc-11e7-8d5a-9086d6ad7b74.png)

</details>
